packages/react-app/src/networks.ts

```diff
@@ -14,11 +14,13 @@ export const networks = Object.freeze({
     chainId: 4,
     name: 'Rinkeby',
     explorerBase: 'etherscan',
-    questsSubgraph: 'https://api.thegraph.com/subgraphs/name/corantin/quests-subgraph',
-    governSubgraph: 'https://api.thegraph.com/subgraphs/name/corantin/govern-1hive-rinkeby',
+    // questsSubgraph: 'https://api.thegraph.com/subgraphs/name/corantin/quests-subgraph',
+    questsSubgraph: 'https://api.thegraph.com/subgraphs/name/corantin/quests-subgraph-staging',
+    // governSubgraph: 'https://api.thegraph.com/subgraphs/name/corantin/govern-1hive-rinkeby',
+    governSubgraph: 'https://api.thegraph.com/subgraphs/name/corantin/govern-1hive-rinkeby-staging',
```

Comment:
Maybe we could put it array in the future to easily change beetween subgraphs dependen of network, wdyt?

Reply:
Its handled like this right now
![image](https://user-images.githubusercontent.com/13890618/168332226-f008a206-1df8-472f-852d-4ead4ce12ea2.png)
![image](https://user-images.githubusercontent.com/13890618/168332249-41800bd3-96bf-4245-8710-58dc1d108990.png)